tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -54,23 +101,26 @@ case "$DATABASE_TYPE" in
     fi
     ;;
 esac
-
-# Initialize database if needed
+loggood "checking of database success"
+# Initialize database if needed
 if [ "$INIT_DATABASE" = true ] ; then
+    log "Initialise Database"
     cd /tracim/backend/
-    tracimcli db init -c /etc/tracim/development.ini
+    tracimcli_as_user "db init"
+    loggood "Database initialisation success"
 else
+    log "Update database"
     cd /tracim/backend/
-    alembic -c /etc/tracim/development.ini upgrade head
+    alembic_as_user "upgrade head"
```

Comment:
some question about `alembic -c` (maybe I just not seen this in a beggining of the file)